tests/Integration/Routing/RouteSingletonTest.php

```diff
@@ -259,7 +259,7 @@ public function testCreatableApiSingletonOnly()
         $this->assertEquals(405, $response->getStatusCode());
 
         $response = $this->get('/avatar/create');
-        $this->assertEquals(404, $response->getStatusCode());
+        $this->assertEquals(200, $response->getStatusCode());
```

Comment:
It doesn't make sense, but it's consistent with how the `apiResource` method works.

The `apiResource` method is just a wrapper around the `resource` method that configures "only" to reduce the default routes. If you call `only` afterwards, you effectively undo the effects of calling `apiResource` instead of just `resource`.

For example, you can call `Route::apiResource('avatars', Foo::class)->only('create')`, and you'll get the `avatars/create` route.

I'm not opposed to changing this behavior, but I think `apiResource` and `apiSingleton` should be consistent with each other, and I don't think we can change the behavior of `apiResource` without a BC.